use-cases/using-SRIOV-and-kata.md

```diff
@@ -0,0 +1,315 @@
+# Setup to use SR-IOV with Kata Containers and Docker*
+
+- [Install the SR-IOV Docker\* plugin](#install-the-sr-iov-docker-plugin)
+- [Host setup for SR-IOV](#host-setup-for-sr-iov)
+	- [Checking your NIC for SR-IOV](#checking-your-nic-for-sr-iov)
+	- [IOMMU Groups and PCIe Access Control Services](#iommu-groups-and-pcie-access-control-services)
+	- [Update the host kernel](#update-the-host-kernel)
+- [Set up the SR-IOV Device](#set-up-the-sr-iov-device)
+- [Example: Launch a Kata Containers container using SR-IOV](#example-launch-a-kata-containers-container-using-sr-iov)
+
+Single Root I/O Virtualization (SR-IOV) enables splitting a physical device into
+virtual functions (VFs). Virtual functions enable direct passthrough to virtual
+machines or containers. For Kata Containers, we enabled a Container Network
+Model (CNM) plugin. Additionally, we made the necessary changes in the
+runtime to detect virtual functions in a container's network namespace to use
+SR-IOV for network based devices.
+
+## Install the SR-IOV Docker\* plugin
+
+To create a network with associated VFs, which can be passed to
+Kata Containers, you must install a SR-IOV Docker plugin. The
+created network is based on a physical function (PF) device. The network can
+create 'n' containers, where 'n' is the number of VFs associated with the
+Physical Function (PF).
+
+To install the plugin, follow the [plugin installation instructions](https://github.com/clearcontainers/sriov).
+
+
+## Host setup for SR-IOV
+
+In order to setup your host for SR-IOV, the following has to be true:
+
+- The host system must support Intel VT-d.
+- Your device (NIC) must support SR-IOV.
+- The host kernel must have Input-Output Memory Management Unit (IOMMU)
+  and Virtual Function I/O (VFIO) support.
+- `CONFIG_VFIO_NOIOMMU` must be disabled in the host kernel
+  configuration. You must rebuild your host system's kernel in
+  order to disable `CONFIG_VFIO_NOIOMMU` in the kernel configuration.
+- Optionally, you might need to add a PCI override for your Network Interface
+  Controller (NIC). The section [Checking your NIC for SR-IOV](#checking-your-nic-for-sr-iov) describes how to assess if you need to make NIC changes and how to make
+  the necessary changes.
+
+### Checking your NIC for SR-IOV
+
+The following is an example of how to use `lspci` to check if your NIC supports
+SR-IOV.
+
+```bash
```

Comment:
Please can you remove `bash` here - it's causing the CI doc checking code to fail as all bash blocks are expected to (only) contain commands the user should run - not also show the output.

See: https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#code-blocks.

Once done,

lgtm.

Reply:
@jodh-intel Done.